packages/build-tools/src/buildErrors/userErrorHandlers.ts

```diff
@@ -256,7 +256,7 @@ To resolve this issue, downgrade to an older Xcode version using the "image" fie
     createError: () =>
       new UserFacingError(
         errors.ErrorCode.UNKNOWN_FASTLANE_ERROR,
-        `Fastlane build failed with unknown error. See logs for the "Run fastlane" and "Xcode Logs" phases for more information.
+        `Fastlane build failed with unknown error. See logs for the "Run fastlane" (it contains filtered build logs) and "Xcode Logs" (it contains full build logs) phases for more information.
```

Comment:
Correct me if I'm wrong but the "Xcode build Logs" doesn't contain full build logs but rather more detailed/different level logs.

Reply:
```suggestion
        `Fastlane build failed with unknown error. See logs for the "Run fastlane" and "Xcode Logs" (especially, if the error message from "Run fastlane" is generic) phases for more information.
```

Reply:
I don't have a good idea how to describe it better, maybe @brentvatne has an idea?